README.md

```diff
@@ -9,6 +9,7 @@ A curated list of awesome open source workflow engines
 * [Argo Workflows](https://github.com/argoproj/argo-workflows) [![Stars](https://img.shields.io/github/stars/argoproj/argo-workflows.svg)](https://github.com/argoproj/argo-workflows) - Open source container-native workflow engine for getting work done on Kubernetes
 * [Azkaban](https://azkaban.github.io/) [![Stars](https://img.shields.io/github/stars/azkaban/azkaban.svg)](https://github.com/azkaban/azkaban) - Batch workflow job scheduler created at LinkedIn to run Hadoop jobs.
 * [Brigade](https://brigade.sh/) [![Stars](https://img.shields.io/github/stars/brigadecore/brigade.svg)](https://github.com/brigadecore/brigade) - Brigade is a tool for running scriptable, automated tasks in the cloud — as part of your Kubernetes cluster.
+* [C++ Workflow](https://github.com/sogou/workflow) [![Stars](https://img.shields.io/github/stars/sogou/workflow.svg)](https://github.com/sogou/workflow) - C++ Parallel Computing and Asynchronous Networking Engine.
```

Comment:
@Barenboim I think this fits more to Lib section https://github.com/meirwah/awesome-workflow-engines#library-embedded-usage no ?

Reply:
No problem. Would you move it to the section for us?

Reply:
Done! Thank you.